src/components/TaskContainer.jsx

```diff
@@ -1,22 +1,35 @@
 import { useSelector } from 'react-redux';
+import { useState } from 'react';
 import AddTask from 'components/AddTask';
 import NoTaskPlaceholder from 'components/NoTaskPlaceholder';
 import Task from 'components/Task';
+import Button from 'components/ui/Button';
+import { paginationLabel } from 'utils/helpers';
+import { nextPage, paginate } from 'utils/helpers/ReducerHelper';
 
 export default function TaskContainer() {
   const tasks = useSelector((state) => state.todoStates.todos);
   const isNewTaskRequested = useSelector((state) => state.todoStates.isNewTaskRequested);
+  const [currentPage, setCurrentPage] = useState(1);
   const isTasksAvailable = tasks.length || isNewTaskRequested;
+  const isPaginationAvailable = tasks.length > 9;
```

Comment:
- [ ] If there is 9 task on the board and you open create task card, Then what will be the scenario? Does the `show more` button showed that time? If not than handle it.
- [ ] If there is more than 18 task, how many times you need to press `show more` button? Do it handling in your PR? if not then handle it.
- [ ] When a task created while showing all the task or `show less` button, Does it reset after that? If not then handle it.

thanks.